lib/manageiq/api/client/collection.rb

```diff
@@ -11,6 +11,25 @@ def initialize(*_args)
           raise "Cannot instantiate a #{self.class}"
         end
 
+        # As per Rails 5:
```

Comment:
I don't think we need the Rails 5 reference...  the rest of the documentation is good though.